src/core/gui/toolbarMenubar/ToolMenuHandler.cpp

```diff
@@ -325,8 +325,8 @@ void ToolMenuHandler::initEraserToolItem() {
 void ToolMenuHandler::signalConnectCallback(GtkBuilder* builder, GObject* object, const gchar* signalName,
                                             const gchar* handlerName, GObject* connectObject, GConnectFlags flags,
                                             ToolMenuHandler* self) {
-    string actionName = handlerName;
-    string groupName{};
+    std::string actionName = handlerName;
+    std::string groupName{};
```

Comment:
What is the purpose of this change? It's not like `using std::string` can be removed after that, since `string` (without `std::`) appears on other places in the file as well.

Reply:
You're quite right here. I removed those changes.